src/App.test.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
 
 test('renders learn react link', () => {
-  render(<App />);
-  const uploadButton = screen.getByText('UPLOAD VIDEO');
+  render(<BrowserRouter><App /></BrowserRouter>);
```

Comment:
never used browserRouter so I cant really give you much feedback on the router design, I trust you did a good job though

Reply:
This was only to shut the error from the test :D